### ISSUE TITLE
We don't return an error for wrong keywords.

### ISSUE THREAD

**Issue by pawanrawal** [dgraph-io/dgraph#534]
```
curl localhost:8080/query -XPOST -d '{
  director(squalof("type.object.name.en", "steven spielberg")) {
    type.object.name.en
    film.director.film (disorder: film.film.initial_release_date) @filter(geq("film.film.initial_release_date", "1984-08")) {
      type.object.name.en
      film.film.initial_release_date
    }

  }
}
' | python -m json.tool | less
```
Even though I have squalof, which should return an error since we don't recognize it, we return results for anyof. Similar if you would have anything except (order | orderdesc), like disorder above, results are returned in a random order. Probably we should be more strict and return errors?

**Comment by ashishnegi** [comment]
Should be fixed with [commit #702f0392](https://github.com/dgraph-io/dgraph/commit/702f03921ddc88e57af5c5d9daca4427e21d927f)

Now we return :
`{"code":"ErrorInvalidRequest","message":"Invalid function name : shillof"}`

**Comment by ashishnegi** [comment]
Verified with @pawanrawal 
Closing the issue.
